src/mmw/js/src/water_balance/models.js

```diff
@@ -31,26 +16,24 @@ var WaterBalanceModel = {
             return Math.round(parseFloat(x) * 10) / 10;
         }
 
-        function processLine(soil, lands, precip) {
+        function processLine(soil, land, precip) {
             if (!model[soil]) {
                 model[soil] = {};
             }
-            lands.forEach(function(land) {
-                if (!model[soil][land]) {
-                    model[soil][land] = {};
-                }
-                model[soil][land][precip] = {
-                    'et': round(line[3]),
-                    'i' : round(line[4]),
-                    'r' : round(line[5])
-                };
-            });
+            if (!model[soil][land]) {
+                model[soil][land] = {};
+            }
+            model[soil][land][precip] = {
+                'et': round(line[3]),
+                'i' : round(line[4]),
+                'r' : round(line[5])
+            };
         }
 
         for (var j = 1; j < lines.length; j++) {
             var line = lines[j].split(',');
             var soil = this.soilMap[line[2]];
-            var lands = this.landMap[line[1]];
+            var lands = line[1];
```

Comment:
Perhaps we can now rename `lands` to `land` since it always has exactly one value.